plugins/parquet/README.md

```diff
@@ -78,4 +78,66 @@ To select the fields that you wish to partition your data on pass in a string of
+| UInt64                      | UNSIGNED8       |       |
+| Float16 (Half Float)        | REAL4           | Stored as REAL4 in ECL; Precision may be lost due to conversion from 16-bit to 32-bit float |
+| Float                       | REAL4           | ECL REAL4 is a 32-bit floating-point number |
+| Double                      | REAL8           | ECL REAL8 is a 64-bit floating-point number |
+| Decimal128, Decimal256      | DECIMAL         | Precision and scale in ECL must match those in Parquet; ECL DECIMAL allows variable precision |
+| Date32, Date64              | UNSIGNED        | Stored as days or milliseconds since epoch; ensure proper date handling in ECL |
+| Time32, Time64              | UNSIGNED        | Stored as seconds or milliseconds since epoch; ECL UNSIGNED type can handle these formats |
+| Timestamp                   | UNSIGNED        | Stored as microseconds since epoch; may need conversion for precise handling in ECL |
+| Duration                    | RECORD          | Custom ECL structure may be needed to represent complex durations (e.g., days, hours, minutes) |
+| String, LargeString         | STRING, VARSTRING, UTF8 | ECL STRING or VARSTRING for variable-length text; UTF8 encoding may be used for compatibility |
+| Binary, LargeBinary         | DATA            | ECL DATA for binary data; handles raw byte storage |
+| FixedSizeBinary             | DATA            | Fixed-length binary data; ensure length matches between Parquet and ECL |
+| List, LargeList             | SET OF          | ECL SET OF to handle collections; `LargeList` corresponds to large collections in ECL |
+| Struct                      | RECORD          | Nested structures in ECL; ensure field definitions match Parquet schema |
+
+Note: When using `ParquetIO.read()`, ensure that the ECL record structure matches the Parquet schema. For complex types like Duration, you may need to define custom ECL structures.
+
+For more detailed information about Apache Arrow data types, refer to the [Apache Arrow Documentation](https://arrow.apache.org/docs/cpp/api/datatype.html).
+
+# Example Usage
+Here's a simple example demonstrating how to read and write Parquet files using basic data types:
+eclCopyIMPORT ParquetIO;
+
+```
+// Define a simple record structure
+PersonRecord := RECORD
+    STRING30 name;
+    UNSIGNED2 age;
+    BOOLEAN isEmployee;
+    REAL4 salary;
+END;
+
+// Sample dataset
+sampleData := DATASET([
+    {'John Doe', 30, TRUE, 50000.50},
+    {'Jane Smith', 28, TRUE, 52000.75},
+    {'Bob Johnson', 45, FALSE, 0.00}
+], PersonRecord);
+
+// Write the dataset to a Parquet file
+ParquetIO.Write(sampleData, '/var/lib/HPCCSystems/mydropzone/employees.parquet');
+
+// Read the Parquet file back into an ECL dataset
+readData := ParquetIO.Read(PersonRecord, '/var/lib/HPCCSystems/mydropzone/employees.parquet');
+
+// Display the data
+OUTPUT(readData);
+
+```
+# Obtaining the Plugin
```

Comment:
This seems like it would make more sense in the installation section. Just add a small note saying the Parquet plugin is included in the base package by default and if they are missing it they can either turn on -DUSE_PARQUET=ON or follow the instructions below.